Makefile

```diff
@@ -25,11 +25,12 @@ STACK_NAME ?= ghostscript-layer
 result/bin/gs: all
 
 build/output.yaml: template.yaml
-	aws cloudformation ${PROFILE} package --template $< --s3-bucket $(DEPLOYMENT_BUCKET) --output-template-file $@
+	rm -f build/output.yaml
+	aws cloudformation package --template $< --s3-bucket $(DEPLOYMENT_BUCKET) --output-template-file $@
 
 deploy: build/output.yaml
-	aws cloudformation ${PROFILE} deploy --template $< --stack-name $(STACK_NAME)
```

Comment:
original one was deploying by passing the profile na from `./aws/credentials` but we don't need that with aws-vault